src/templates/layouts/healthCareLocalFacility/Phone.tsx

```diff
@@ -0,0 +1,63 @@
+import { PhoneNumber, separateExtension } from '@/templates/common/phoneNumber'
+import { ParagraphPhoneNumber } from '@/types/drupal/paragraph'
+import { formatter as formatParagraphPhoneNumber } from '@/data/queries/phoneNumber'
+
+export const processPhoneToVaTelephoneOrFallback = (
+  phoneNumber: string = ''
+) => {
+  const internationalPattern = /\(?(\+1)\)?[- ]?/gi
+
+  if (!phoneNumber) {
+    return null
+  }
+
+  const separated = separateExtension(phoneNumber)
+
+  if (separated.processed) {
+    return (
+      <va-telephone
+        contact={separated.phoneNumber}
+        extension={separated.extension}
+        international={internationalPattern.test(phoneNumber)}
+      />
+    )
+  }
+
+  return <a href={`tel:+1${phoneNumber}`}>{phoneNumber}</a>
```

Comment:
This should still use the va-telephone web-component
```suggestion
  return <va-telephone contact={phoneNumber} />
```

Reply:
Ah, good call, good call. Just verified the behavior is still what we want, and it is. 👍